client/src/legacy/GridField.js

```diff
@@ -19,9 +19,13 @@ $.entwine('ss', function($) {
         this.fixColumns();
         this.injectSearchButton(false);
       }
+      
+      if (this.hasFilters()) {
+        this.injectSearchButton(true);
+      };
 
       if (this.is('.grid-field--lazy-loadable') && (
-        (this.closest('.ss-tabset').length === 0) || (this.data('gridfield-lazy-load-state') === 'force') )
+        (this.closest('.ss-tabset, .cms-tabset').length === 0) || (this.data('gridfield-lazy-load-state') === 'force') )
```

Comment:
Sorry, probably it appeared after rebase. I can remove it, but it exist in branch 1 already, just in case.

Reply:
Yeah in the `1` branch it's there to support a change in lazy-loading behaviour. When you retarget the `1` branch it will be there but won't count as a change so should be fine once you swap to targetting `1`.